Dockerfile

```diff
@@ -8,7 +8,8 @@ COPY docker-entrypoint.sh ${JB_HOME}
 COPY theme/akvo/ ${KC_HOME}/themes/akvo/
 COPY keycloak_configuration.txt ${KC_HOME}
 
-RUN ${KC_HOME}/bin/jboss-cli.sh --file=${KC_HOME}/keycloak_configuration.txt && \
+RUN ${KC_HOME}/bin/jboss-cli.sh --file=${KC_HOME}/keycloak_configuration.txt | tee ${KC_HOME}/cli-result.txt && \
+    grep outcome ${KC_HOME}/cli-result.txt | grep -v success && exit 123 || \
```

Comment:
Hi, let's use `set -e` at the beginning of the `RUN` directive. We don't need to check manually that the result of applying a configuration works, as a failure returns `exit 1`.

My tests.
````
[jboss@58bf74f8f62f keycloak]$ bin/jboss-cli.sh --file=/code/keycloak_configuration.txt 
08:09:13,565 INFO  [org.jboss.modules] (main) JBoss Modules version 1.6.1.Final
08:09:13,724 INFO  [org.jboss.msc] (main) JBoss MSC version 1.2.7.SP1
08:09:13,908 INFO  [org.jboss.as] (MSC service thread 1-7) WFLYSRV0049: Keycloak 3.4.3.Final (WildFly Core 3.0.8.Final) starting
08:09:16,198 INFO  [org.jboss.as.controller.management-deprecated] (Controller Boot Thread) WFLYCTL0028: Attribute 'security-realm' in the resource at address '/core-service=management/management-interface=http-interface' is deprecated, and may be removed in future version. See the attribute description in the output of the read-resource-description operation to learn more about the deprecation.
08:09:16,343 INFO  [org.wildfly.security] (Controller Boot Thread) ELY00001: WildFly Elytron version 1.1.6.Final
08:09:16,364 INFO  [org.jboss.as.controller.management-deprecated] (Controller Boot Thread) WFLYCTL0028: Attribute 'security-realm' in the resource at address '/subsystem=undertow/server=default-server/https-listener=https' is deprecated, and may be removed in future version. See the attribute description in the output of the read-resource-description operation to learn more about the deprecation.
08:09:16,653 INFO  [org.jboss.as.patching] (MSC service thread 1-7) WFLYPAT0050: Keycloak cumulative patch ID is: base, one-off patches include: none
08:09:16,671 WARN  [org.jboss.as.domain.management.security] (MSC service thread 1-5) WFLYDM0111: Keystore /opt/jboss/keycloak/standalone/configuration/application.keystore not found, it will be auto generated on first use with a self signed certificate for host localhost
08:09:16,792 INFO  [org.jboss.as.server] (Controller Boot Thread) WFLYSRV0212: Resuming server
08:09:16,794 INFO  [org.jboss.as] (Controller Boot Thread) WFLYSRV0025: Keycloak 3.4.3.Final (WildFly Core 3.0.8.Final) started in 3219ms - Started 64 of 83 services (30 services are lazy, passive or on-demand)
{"outcome" => "success"}
{"outcome" => "success"}
{"outcome" => "success"}
{"outcome" => "success"}
{"outcome" => "success"}
{"outcome" => "success"}
{"outcome" => "success"}
{"outcome" => "success"}
{"outcome" => "success"}
{"outcome" => "success"}
{"outcome" => "success"}
{"outcome" => "success"}
{"outcome" => "success"}
{"outcome" => "success"}
{
    "outcome" => "failed",
    "failure-description" => "WFLYCTL0369: Required capabilities are not available:
    org.wildfly.clustering.jgroups.channel-factory.${env.JGROUPS_STACK:tcp}; Possible registration points for this capability: 
		/subsystem=jgroups/channel=*
		/subsystem=jgroups/channel=*/fork=*
		/subsystem=jgroups/stack=*",
    "rolled-back" => true
}
08:09:17,376 INFO  [org.jboss.as] (MSC service thread 1-7) WFLYSRV0050: Keycloak 3.4.3.Final (WildFly Core 3.0.8.Final) stopped in 21ms

[jboss@58bf74f8f62f keycloak]$ echo $?
1
````